src/LinearSolvers/iterativeWrapper.jl

```diff
@@ -24,6 +24,7 @@ Example:
 
 """
 type IterativeSolver<: AbstractSolver
+   AA   # matrix being solved
```

Comment:
When using the direct solver, most of the memory is used to store the factorization.  However, the memory needed to store the PDE is also significant, and I think it is best to deallocate it as soon as it is no longer needed.